SU-SWS/saa_victoria_subtheme
# ADAPT-453 Fix button and cta link styles

# READY FOR REVIEW

# Summary
- Fixes banner button styles
- Fixes CTA link icon color issue
- Fixes accessibility issues with link within the testimonial and footer

# Review By (Date)
- 07/16

# Urgency
- low

# Steps to Test
1. Pull this to local
2. Clear cache
3. Create a Basic page with top banner and in-page banner with no image. Include all button variations for testing. The top banner should have buttons using the `su-button-secondary` style. The in-page banner should display the normal button styles.

# Associated Issues and/or People
- [ADAPT-453](https://stanfordits.atlassian.net/browse/ADAPT-453)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
